doc/modules/partial_dependence.rst

```diff
@@ -145,13 +145,13 @@ ICE plots:
 
```

Comment:
l.141, there is an occurence for the function `plot_partial_dependence`. We should instead refer to the class method `.from_estimator`